src/engine/irc.cpp

```diff
@@ -360,19 +360,31 @@ ICOMMAND(0, ircbind, "ss", (const char *name, const char *s), {
 ICOMMAND(0, ircpass, "ss", (const char *name, const char *s), {
     ircnet *n = ircfind(name);
     if(!n) { conoutf("no such ircnet: %s", name); return; }
-    if(!s || !*s) { ircprintf(n, 4, NULL, "current password is: %s", n->passkey && *n->passkey ? "<set>" : "<not set>"); return; }
+    if(!s || !*s)
+    {
+        ircprintf(n, 4, NULL, "current password is: %s", *n->passkey ? "<set>" : "<not set>");
```

Comment:
You just reformatted here, right? Same thing in these if clauses, would appreciate if you could make pointer stuff explicit.

Reply:
not just formating, I also removed `n->passkey && ` from `n->passkey && *n->passkey` cuz compiler said address will always return true (make sense).

Reply:
Can you please post the compiler warning here completely? I'm curious.

Reply:
probably `tautological-type-limit-compare`

Reply:
`address of array 'c->passkey' will always evaluate to 'true' [-Wpointer-bool-conversion]GCC`